wagtail/admin/static_src/wagtailadmin/scss/layouts/page-editor.scss

```diff
@@ -607,6 +607,11 @@ footer .preview {
       @include column(10);
       padding-inline-start: 0;
       padding-inline-end: 0;
+
+      > li {
+        padding-inline-end: 1.25rem;
+        padding-inline-start: 1.25rem;
```

Comment:
```suggestion
        padding: 1em 10em 1em 1.5em;
```

Otherwise we still end up with a broken UI:

![inlinepanel-padding](https://user-images.githubusercontent.com/877585/165940966-18100204-6ff7-4ac4-a87f-8eb7824ac039.png)